packages/indicators/src/Builder/CountEventBuilder/config/types.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+/**
+ * Default Value for event data values
+ * 'undefined' can be passed in to treat the value as "optional"
+ */
+export type DefaultValue = number | string | 'undefined';
+
+/**
+ * A flexible config schema that can be provided as an input to the indicator
+ */
+export type CountEventConfig = {
+  readonly formula: string;
+  readonly programCode: string;
+  readonly defaultValues?: Record<string, DefaultValue>;
+};
```

Comment:
This is the spec of `countEvent` config. We don't have `aggregation` like `arithmetic` indicator because we usually don't aggregate event data. I've also left `paramBuilders` out because:

1. We don't need it at the moment
2. I feel that it's better to wait until there's specific use case for it so that we can come up with a more concise config format?